platforms/corda-ent/configuration/samples/network-cordaent.yaml

```diff
@@ -5,7 +5,7 @@ network:
   # Network level configuration specifies the attributes required for each organization
   # to join an existing network.
   type: corda-enterprise
-  version: 4.4
+  version: 1.2
```

Comment:
is 4.4 not 1.2 1.2 is only CENM Version

Reply:
ok we can change that, but for cenm we have 1.3 version as well right?

Reply:
have changed this